Mods/BetterMachines/src/BetterMachines/BetterMachinesPlugin.cs

```diff
@@ -383,35 +383,36 @@ private void Awake()
                 ProtoRegistry.AddLodMaterials("Assets/BetterMachines/Machines/chemical-plant-mk3", 1, new[] {chemicalPlantMatMk3Lod, chemicalPlantMatGlass});
                 ProtoRegistry.AddLodMaterials("Assets/BetterMachines/Machines/chemical-plant-mk3", 2, new[] {chemicalPlantMatMk3Lod, chemicalPlantMatGlass});*/
 
-                ProtoRegistry.RegisterModel(453, storageMk3Item, "Assets/BetterMachines/Machines/storage-3",
-                    new[] {storageMat}, new[] {17, 1}, 403);
+                ProtoRegistry.RegisterModel(23, storageMk3Item, "Assets/BetterMachines/Machines/storage-3",
```

Comment:
Why is this model id so low? Considering others are in 600 ish range, this looks suspicious.